src/css/profile/mobile/common/masteronoffswitch.less

```diff
@@ -0,0 +1,32 @@
+.ui-master-on-off-switch {
+    .font(regular);
+    font-size: 18 * @sp_base;
+    margin-left: 0;
+    margin-right: 0;
+    margin-bottom: 20 * @px_base;
```

Comment:
Ditto. margin: 0 0 20*@px_base 0;

Reply:
Hi, but I don't want to change margin-top to "0"

Reply:
If you don't want to change the top value, then you can use the "inherit"?
Anyway,, it is not important point :(

Reply:
Sometimes these redundant styles make issues in different place of framework because the selectors are too strong and we cannot use "important" in framework.